resources/authgear/templates/en/web/authflowv2/whatsapp_otp.html

```diff
@@ -16,6 +16,19 @@
 		{{ end }}
 	{{ end }}
 
+  {{ if $.Error }}
+    {{ if not (eq $.Error.reason "InvalidCredentials") }}
```

Comment:
Noticed that whatsapp and oob otp has different error reason, is it intended?

Reply:
I think both reasons could happen in both cases, depends on the purpose of the otp is verification or authentication?

Reply:
https://github.com/authgear/authgear-server/blob/main/pkg/lib/authenticationflow/declarative/node_verify_claim.go#L178

Reply:
Should cover both reasons for the pages then, let me update that 🙏